### ISSUE TITLE
[Feature request] Highlight imported GPX-track in the map

### ISSUE THREAD

**Issue by update-freak** [organicmaps/organicmaps#7777]
**Is your feature request related to a problem? Please describe.**
I have imported many GPX-tracks and all are selected to show.
Therefore at some regions the tracks are overlapping and it's not clear which is the correct path for a speific track.

![organicmaps](https://github.com/organicmaps/organicmaps/assets/40196995/8669981e-e1be-48da-927b-4c34c4b02fda)

**Describe the ideal solution**
I think it makes sense to add the posibility to click at a position of the imported track that the complete tour is highlighted. Maybe also a button with the name and an option to hide should appear.

Proposal
![organicmaps proposal](https://github.com/organicmaps/organicmaps/assets/40196995/9120cbc5-35fe-4f3c-b405-edd04a8b295e)

In Komoot for example it's solved like this with different colors: 
![komoot](https://github.com/organicmaps/organicmaps/assets/40196995/805a615c-b868-498a-9fcb-40b388512214)

**Describe alternatives you have considered**
Alternative would be to hide every track and one by one enable them, but this is not prefered.

**Comment by biodranik** [comment]
The highlighting is also needed for route relations.

**Comment by Rob-from-VI** [comment]
Yes please highlight and bring to front is much needed. Directional arrows would also be great, for figure 8 routes.
